### DISCUSSION TITLE
Information

### DISCUSSION THREAD

**Discussion by Schtailian** [target/goalert#2194] [Ideas]
Hi,
we have a use case where it would be nice to always inform some people of an alert, independent of the escalation policy. For example, escalate to the on-call people but also inform management of an ongoing issue.

![image](https://user-images.githubusercontent.com/10855460/155482226-01bcc995-6e90-4851-bf62-afe294d59e50.png)
 just a little sketch, what i was thinking about

It would also be nice, to override the notification rules per policy per user. Or is that already possible

Let me know what you think about that.

**Comment by Lanithane** [comment]
Hey Schtailian, sorry that I'm just now seeing this! We haven't explored Github Discussions much as a team yet.

I think for this use case, to always inform these people you could add them to step 1, as step 1 will always be the first path taken for alerting- unless I'm misunderstanding?
